src/em/prov/easyconnect/ec_ctrl_configurator.cpp

```diff
@@ -942,6 +942,7 @@ std::pair<uint8_t *, size_t> ec_ctrl_configurator_t::create_recfg_auth_confirm(s
     uint8_t trans_id = 0;
     ec_dpp_reconfig_flags_t reconfig_flags = {
         .connector_key = 1, // DONT REUSE
+        .reserved = 0,
     };
```

Comment:
```suggestion
    ec_dpp_reconfig_flags_t reconfig_flags;
    memset(&reconfig_flags, 0, sizeof(ec_dpp_reconfig_flags_t));
    reconfig_flags.connector_key = 1;
```
@amarnathhullur  I believe this should resolve both the warning and the openwrt based build issue

Reply:
yes probably this should fix.